src/core/redux/Store.tsx

```diff
@@ -19,6 +20,7 @@ export const initStore = (initialState: {} = {}) => {
       jobTypes: jobTypesReducer,
       payments: paymentsReducer,
       shop: shopReducer,
+      events: eventsReducer,
```

Comment:
Don't know if its worth keeping to do it, but i tried to sort the reducer names alphabetically

Reply:
Wouldn't it be easier to sort this through linting, like https://eslint.org/docs/rules/sort-keys, og https://palantir.github.io/tslint/rules/object-literal-sort-keys/?

Reply:
I don't think we should turn it on for the whole project, but this object will be huge, and I think it is beneficial just here.

Is it possible to selectively turn linting rules on for specific parts of code?

Reply:
Yes you can https://palantir.github.io/tslint/usage/rule-flags/
syntax:
```js
/* tslint:enable:rule1 rule2 */
{ code... }
/* teslint:disable: rule1 rule2 */
```
I don't think there is a way to reset it back to default value from config